src/linear_algebra/invariant_basis_number.lean

```diff
@@ -96,6 +97,24 @@ begin
     ((P.injective.comp i).comp Q.injective)
 end
 
+/-- A ring satisfies the strong rank condition if and only if, for all `n : ℕ`, there are no
+injective linear maps `(fin (n + 1) → R) →ₗ[R] (fin n → R)`. -/
+lemma strong_rank_condition_iff : strong_rank_condition R ↔
+  ∀ (n : ℕ), ¬∃ f : (fin (n + 1) → R) →ₗ[R] (fin n → R), function.injective f :=
```

Comment:
I get the feeling this statement would be more useful as
```suggestion
  ∀ (n : ℕ) (f : (fin (n + 1) → R) →ₗ[R] (fin n → R)), ¬function.injective f :=
```
as forall is easier to use than not exists.